migrations/20220421011156_initial.ts

```diff
@@ -60,9 +60,8 @@ export async function up(knex: Knex): Promise<void> {
         table.smallint(DB_CONSTANTS.BOOKS.BOOK_TYPE).notNullable();
     })
 
-    knex.schema.createTable(DB_CONSTANTS.PROMPTS.TABLE, function(table){
+    await knex.schema.createTable(DB_CONSTANTS.PROMPTS.TABLE, function(table){
         table.increments();
-        table.string(DB_CONSTANTS.PROMPTS.GUILD_ID);
```

Comment:
I removed this line because `knex` was erroring out from it. I'm unsure if that was the right thing to do, but it looked like a mistake given the line below.